gson/src/main/java/com/google/gson/GsonBuilder.java

```diff
@@ -632,6 +635,28 @@ public GsonBuilder disableJdkUnsafe() {
     return this;
   }
 
+  /**
+   * Adds a reflection access filter. A reflection access filter prevents Gson from using
+   * reflection for the serialization and deserialization of certain classes. The logic in
+   * the filter specifies which classes those are.
+   *
+   * <p>Filters will be invoked in reverse registration order, that is, the least recently
```

Comment:
Isn't this just registration order? I would expect "registration order" to mean that the earliest one is invoked first.

Reply:
Yes, you are right, thanks for pointing that out!